01/1_1.playground/Contents.swift

```diff
@@ -0,0 +1,79 @@
+/*
+Exercise 1.1.  Below is a sequence of expressions. What is the result printed by the interpreter in response to each expression? Assume that the sequence is to be evaluated in the order in which it is presented.
+*/
+
+
+/*
+10
+*/
+10
+
+/*
+(+ 5 3 4)
+*/
+5 + 3 + 4
+
+/*
+(- 9 1)
+*/
+9 - 1
+
+/*
+(/ 6 2)
+*/
+6 / 2
+
+/*
+(+ (* 2 4) (- 4 6))
+*/
+(2 * 4) + (4 - 6)
+
+/*
+(define a 3)
+*/
+var a = 3
+
```

Comment:
I think the `scheme` implementation gives us a mutable object here, which is why I chose `var` when I ported to swift. I am no `scheme` connoisseur, however. @jwatson ? @klundberg ?